packages/flutter_tools/templates/app/windows.tmpl/runner/win32_window.h

```diff
@@ -52,6 +52,9 @@ class Win32Window {
   // If true, closing this window will quit the application.
   void SetQuitOnClose(bool quit_on_close);
 
+  // Return a RECT representing the bounds of the current window.
```

Comment:
This comment and name are pretty vague; there are different things that it could refer to (e.g., I would have guessed without having seen the implementation that it was the exterior bounds). How about `GetClientArea`? (I don't think we need `Window` in the name of a method on a window object; it's implied.)

Reply:
OK, will rename to GetClientArea

Reply:
qarni

Reply:
Fixed in more recent commit